src/openforms/js/components/admin/form_design/logic/actions/dmn/DMNActionConfig.stories.js

```diff
@@ -232,10 +233,16 @@ export const Empty = {
 
       const [formVarsDropdowns, dmnVarsDropdown] = dropdowns;
 
-      await userEvent.selectOptions(formVarsDropdowns, 'Name (name)');
       await userEvent.selectOptions(dmnVarsDropdown, 'camundaVar');
+      await selectEvent.select(formVarsDropdowns, (content, element) => {
+        return (
+          content === '(name)' && element.classList.contains('form-variable-dropdown__option__key')
+        );
+      });
 
-      await expect(formVarsDropdowns.value).toBe('name');
+      await expect(
+        formVarsDropdowns.closest('.form-variable-dropdown').querySelector('input[type="hidden"]')
+      ).toHaveValue('name');
       await expect(dmnVarsDropdown.value).toBe('camundaVar');
```

Comment:
this should be an `expect(...).toHaveValue(...)` assertion like in the other places. Another way is to use the parent/container and make a `toHaveFormValues(...)` assertion

Reply:
Changed it 👍